### DISCUSSION TITLE
威锋网 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#132] [模板讨论]
威锋网 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by miaoxiaozhao** [comment]
找不到x-access-token，能否用cookies签到？

**Reply by wjf0214** [reply]
更新了，用cookie。

**Comment by miaoxiaozhao** [comment]
Failed at 2/4 request,
Fail assert: {"re": "先登录", "from": "content"} from failed_asserts,
Request URL: https://api.wfdata.club/v1/attendance/userSignIn

今天开始的，更新了cookie也没用。

**Reply by wjf0214** [reply]
我这没问题啊。你是不是cookie有问题。里面有accessToken信息吗？

**Comment by miaoxiaozhao** [comment]
现在又可以了。

**Comment by miaoxiaozhao** [comment]
Failed at 2/4 request,
Fail assert: {"re": "先登录", "from": "content"} from failed_asserts,
Request URL: https://api.wfdata.club/v1/attendance/userSignIn

cookie里有accesstoken，但还是签不上

**Reply by wjf0214** [reply]
更新下cookie再试试，有效期我之前2周。如果更新后可以了，那就是失效了。

**Comment by miaoxiaozhao** [comment]
不行，秒提示上面的错误，刚一点【执行】，1秒不到就弹出上面的错误了。

**Comment by hecarli555** [comment]
请问一下这个签到能不能改成账号密码登录？现在这个ck登录的每个月都会失效，得重新抓ck好繁琐
